### ISSUE TITLE
Create Checkbox Konva Component

### ISSUE THREAD

**Issue by brauliodiez** [Lemoncode/quickmock#21]
** Pending on trasnformer implementation to be merged **

Create a checkbox konva component

<img width="154" alt="image" src="https://github.com/Lemoncode/quickmock/assets/1457912/f3154767-61d7-478f-8352-4e9338900825">

We have got the following checkbox SVG component:

https://github.com/brauliodiez/konba-playground/blob/main/public/widgets/checkbox.svg?short_path=315f416

Implement a React.konba component that takes into consideration resizing, take a look to the combobox konba implementation on Quick Mock

https://github.com/Lemoncode/quickmock/blob/main/src/common/components/front-components/combobox-shape.tsx

Add a solid white background to the input inner content.

Right now we will support dynamic resizing, but we won't bother about adding a text property.

A possible starting point:

```tsx
import React, { forwardRef } from 'react';
import { Group, Rect, Line, Text } from 'react-konva';
import { ShapeConfig } from 'konva/lib/Shape';

interface CheckBoxShapeProps extends ShapeConfig {
  id: string;
  x: number;
  y: number;
  width: number;
  height: number;
  onSelected: (id: string) => void;
}

export const CheckBoxShape = forwardRef<any, CheckBoxShapeProps>(
  ({ x, y, width, height, id, onSelected, ...shapeProps }, ref) => {
    const handleClick = () => {
      onSelected(id);
    };

    return (
      <Group x={x} y={y} ref={ref} {...shapeProps} onClick={handleClick}>
        {/* Caja del checkbox */}
        <Rect
          x={0}
          y={0}
          width={width}
          height={height}
          cornerRadius={5}
          stroke="black"
          strokeWidth={2}
          fill="none"
        />
        
        {/* Marca de verificación (checked) */}
        <Line
          points={[
            width * 0.2, height * 0.5,
            width * 0.4, height * 0.7,
            width * 0.8, height * 0.3
          ]}
          stroke="black"
          strokeWidth={2}
          lineCap="round"
          lineJoin="round"
        />
        
        {/* Texto */}
        <Text
          x={width + 10}
          y={height / 2}
          text="Check me!"
          fontFamily="Comic Sans MS, cursive"
          fontSize={width * 0.8}
          fill="black"
          verticalAlign="middle"
        />
      </Group>
    );
  }
);

export default CheckBoxShape;
```

About styling we will create a separate task to extract common styles for all components (e.g. font used)
